packages/pds-ember/addon/components/pds/checkbox-field/index.js

```diff
@@ -34,14 +34,15 @@ export default class PdsCheckboxField extends Component {
   get id() {
     // use @id arg, if provided
     if (this.args.id) {
-      return this.args.id
+      return this.args.id;
     }
 
     // otherwise, generate and memoize a guid
     if (!this._id) {
-      let guid = guidFor(this)
-      this._id = `chk-${guid}`
+      let guid = guidFor(this);
+      // eslint-disable-next-line ember/no-side-effects
+      this._id = `chk-${guid}`;
```

Comment:
There's a unique-id helper in the works somewhere that might help here (probably a future PR thing), we built our own until its available in core:

https://github.com/hashicorp/consul/blob/cc5dba1e3b88cc670e75b1fafaf777536962e8c6/ui/packages/consul-ui/app/helpers/unique-id.js

We inject our dom service, but you could just use `guidFor` straight in your helper if that feels better for you.

You could just `{{or @id (unique-id)}}` then.